Tests/UITests/TestAppUITests/TestAppLLMOpenAIUITests.swift

```diff
@@ -17,7 +17,7 @@ class TestAppLLMOpenAIUITests: XCTestCase {
         continueAfterFailure = false
         
         let app = XCUIApplication()
-        app.launchArguments = ["--mockMode"]
+        app.launchArguments = ["--mockMode", "--resetKeychain"]
```

Comment:
As I'm just noticing that: Would make sense to add this flag to the schema, by default disabled, as well

Reply:
Great idea - will also change the flag to `--resetSecureStorage` so it is clearer how it works.